docs/en/developement/setup.md

```diff
@@ -12,7 +12,7 @@ Before we start talking about how to set up the environment, we need to do some
 have installed the following software:
 
 * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed.
-* [Java](https://www.java.com/en/download/) (only JDK 8 supported by now) installed and `JAVA_HOME` set.
+* [Java](https://www.java.com/en/download/) ( JDK8/JDK11 are supported by now) installed and `JAVA_HOME` set.
 * [Scala](https://www.scala-lang.org/download/2.11.8.html) (only scala 2.11.8 supported by now) installed.
```

Comment:
You seem to have forgotten the update here

Reply:
Thanks, update the doc.